verticapy/ml_pipeline/README.md

```diff
@@ -0,0 +1,224 @@
+# HEADER
```

Comment:
This file should be added directly to the documentation. You should also write a quick description in the main README.md